### PR TITLE
Adding 4.86.0 and 4.86.0-inprocess

### PR DESCRIPTION
**Pull request by soninaren**
Diff gist: https://gist.github.com/soninaren/cee2bcc99fc9a92d53024b1a1177a1fd/revisions

### CODE REVIEW

**Comment by kshyju** [comment]
Would it add value and help reviewers gain clarity if we updated the PR descriptions with more details about the changes? Specifically, I noticed we're updating the NuGet packages and CLI zip versions. Including links to any relevant work items that triggered these changes might provide better context for the reviewers.

**Comment by soninaren** [comment]
> Would it add value and help reviewers gain clarity if we updated the PR descriptions with more details about the changes? Specifically, I noticed we're updating the NuGet packages and CLI zip versions. Including links to any relevant work items that triggered these changes might provide better context for the reviewers.

It would help for sure to set context. We would have to update the automation to add those details to PR description as most of these reviews are generate via devops pipeline.

File: cli-feed-v4.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
       "hidden": false
     },
     "v0-prerelease": {
-      "release": "4.84.0-inprocess",
+      "release": "4.86.0-inprocess",
```
Comment (surgupta-msft) [inline cli-feed-v4.json:59]: Do we want to skip `4.85-inprocess` here? Checking if that's the process we want to follow.

File: cli-feed-v4.json [hunk 2]
```diff
@@ -27737,6 +27737,415 @@
...
+              "netFrameworkVersion": "v6.0"
+            }
+          }
+        }
+      },
+      "coreTools": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/4.0.6231/Azure.Functions.Cli.linux-x64.4.0.6231.zip",
```
Comment (surgupta-msft) [inline cli-feed-v4.json:27994]: From where are we getting these values? I could not find version 6231 here? https://github.com/Azure/azure-functions-core-tools/releases

Comment (soninaren) [reply cli-feed-v4.json:27994]: It is not released yet. This PR is adding pre-release binaries so we can do OGF tests and then eventually release core tools.

File: cli-feed-v4.json (hunk 1 above)
Comment (soninaren) [reply cli-feed-v4.json:59]: Release tags are not related to core tools or host versions. They are just used internally. Making this change so the numbers match, making it easier to review and spot if anything has gone out of sync

**Review by surgupta-msft — APPROVED** [review]

**Pull request merged**